StitchCore/Tests/stitch-ios-sdk-protoTests/Internal/Common/SynchronizedTests.swift

```diff
@@ -15,16 +15,16 @@ class SynchronizedTests: XCTestCase {
         Thread {
             for i in 0 ..< 100 {
                 queue.addOperation {
-                    try! sync(self) {
-                        let value = counter.value
-                        if i % 2 == 0 {
-                            usleep(50)
-                        }
-                        counter.value += 1
-                        XCTAssert(value == counter.value - 1)
+                    objc_sync_enter(self)
```

Comment:
[minor] Actually do we even need this test here anymore? Don't think this covers anything anymore except Foundation itself.

Reply:
Sort of a sanity check for me. Doesn't matter to me.